style.css

```diff
@@ -1,1990 +1,4132 @@
-  height: 79%;
-  position: absolute;
-  top: 30px;
-  left: 0;
-  border-top-left-radius: 5rem;
-  z-index: -1;
-}
-
-.footer {
-  position: relative;
-  top: 50px;
-  width: 375px;
-  height: 850px;
-  left: -9px;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  z-index: 10;
-  background: url(Images/Contactme.png);
-  background-position: 50% 50%;
-  background-size: cover;
-}
-
-.contact {
-  position: absolute;
-  width: 801px;
-  height: 56px;
-  left: 50px;
-  top: 80px;
-
-  /* Header / 2 */
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-
-  /* or 130% */
-  display: flex;
-  align-items: center;
-  text-align: center;
-
-  /* Neutrals/N0 */
-  color: #fff;
-}
-
-.info {
-  position: absolute;
-  width: 327px;
-  height: 102px;
-  left: 18px;
-  top: 150px;
-
-  /* Header / 2 */
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 300;
-  font-size: 20px;
-  line-height: 28px;
-
-  /* or 130% */
-  text-align: center;
-
-  /* Neutrals/N0 */
-  color: #fff;
-}
-
-input[type=text] {
-  position: relative;
-  width: 75%;
-  border-radius: 8px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  margin: 10px 0;
-  height: 48px;
-  padding: 0 1rem;
-  top: 310px;
-  left: 20px;
-}
-
-input[type=email] {
-  position: relative;
-  width: 75%;
-  border-radius: 8px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  margin: 10px 0;
-  height: 48px;
-  padding: 0 1rem;
-  top: 310px;
-  left: 20px;
-}
-
-textarea {
-  position: relative;
-  width: 75%;
-  border-radius: 8px;
-  background: #fff;
-  border: 1px solid #cfd8dc;
-  margin-top: 8px;
-  padding: 1rem;
-  resize: none;
-  top: 310px;
-  left: 20px;
-}
-
-button {
-  width: 140px;
-  height: 48px;
-  padding: 12px 16px;
-  cursor: pointer;
-  margin: 18px 0;
-  border-radius: 8px;
-  align-items: center;
-  display: flex;
-  flex-direction: row;
-  position: relative;
-  border-color: #fff;
-  top: 330px;
-  left: 20px;
-}
-
-button:focus {
-  background: #2230d2;
-  color: #fff;
-}
-
-button:hover {
-  background: #6070ff;
-  color: #fff;
-}
-
-.btn {
-  /* Button /Normal */
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-
-  /* identical to box height, or 141% */
-
-  display: flex;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-
-  /* Primary/B400 */
-  color: #6070ff;
-  background: #fff;
+@media only screen and (max-width: 768px) {
+  body {
+    display: flex;
+    position: absolute;
+    width: 100%;
+    height: 5657px;
+    left: -12px;
+    top: 0;
+    bottom: 20px;
+    background: #fff;
+  }
+
+  .main {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 406px;
+    height: 4500px;
+  }
+
+  .header {
+    display: flex;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 44px;
+    bottom: 0;
+  }
```

Comment:
- [ ] Please remove the space at the top of your mobile header to match the figma template.
## Your design
![d2](https://user-images.githubusercontent.com/94551484/186763895-e983da92-4345-4b4a-99bc-a5ad5f2c2356.png)
## Figma template
![aaa](https://user-images.githubusercontent.com/94551484/186763928-f3f906b3-77c0-4c06-8659-5c0a02b160e3.png)